quickwit/quickwit-actors/src/actor_context.rs

```diff
@@ -96,6 +89,17 @@ impl<A: Actor> ActorContext<A> {
         }
     }
 
+    pub fn spawn_ctx(&self) -> &SpawnContext {
+        &self.spawn_ctx
+    }
+
+    pub async fn sleep(&self, duration: Duration) {
```

Comment:
nitpick: the function name `sleep` can be misleading, I would stick to `simulate_sleep`.

Reply:
I prefer to stick to sleep here... but comment.